configure.ac

```
@@ -248,6 +248,8 @@ AS_IF([test "x$enable_criu" != "xno"], [
 	        AC_MSG_NOTICE([CRIU version doesn't support join-ns API])])
 	PKG_CHECK_MODULES([CRIU_PRE_DUMP], [criu > 3.16.1], [have_criu_pre_dump="yes"], [have_criu_pre_dump="no"
 	        AC_MSG_NOTICE([CRIU version doesn't support for pre-dumping])])
+	PKG_CHECK_MODULES([CRIU_NETWORK_LOCK], [criu > 3.19], [have_criu_network_lock="yes"], [have_criu_network_lock="no"
```

```suggestion
	PKG_CHECK_MODULES([CRIU_NETWORK_LOCK], [criu >= 3.19], [have_criu_network_lock="yes"], [have_criu_network_lock="no"
```

it's available in 3.19 -- https://github.com/checkpoint-restore/criu/commit/c97cc6a6ce9094ad78ccdaa3a174b5ff7ee07a11

---

thanks, fixed.

I'll merge once the CI is green